### PR TITLE
Update Shapely to 1.5.11

### PR DESCRIPTION
**Pull request by ocefpaf**
Based on https://github.com/ioos/conda-recipes/pull/385.  It works here locally for Python 27 to 35.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@pelson same failure as #105.  

Could  the difference be in out [Dockerfile](https://github.com/ocefpaf/centos64_conda_obvious-ci/blob/master/Dockerfile)?  Our `geos` are virtually the same.  I will take a second look at this later on today.

**Comment by ocefpaf** [comment]
> Could the difference be in out Dockerfile? 

Guess not.  Travis failed too... :unamused:

File: geos/meta.yaml [hunk 1]
```diff
@@ -8,8 +8,7 @@ source:
     - cmake.win.patch  # [win]
 
 build:
-  number: 3  # [win]
-  number: 2  # [not win]
+  number: 3
```
Comment (pelson) [inline geos/meta.yaml]: ooo, ouch. Not sure this compiles in time for the CI services does it? I always thought we would have to build this by hand when it came to GEOS.

**Comment by ocefpaf** [comment]
Bumping `geos` build number on `Linux` and `OSX` seems to have fixed it.

The failures are due to Python 3.5.  I guess the default channel is still  being populated with py35 packages.

File: geos/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline geos/meta.yaml]: Maybe the slower `AppVeyor` might choke, but Circle and Travis finished building `geos` just fine.

PS:  Do you want separate build numbers for Windows and *nix?

**Comment by pelson** [comment]
Thanks @ocefpaf!

**Pull request merged**